plugins/emitevent.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: emitevent
```

Comment:
We often recommend people to split words using - (kebab-case, see Naming doc). 

@corneliusweig wdyt about this?

Reply:
IMO, kebab-case is nicer to read, but harder to type. We are not completely consistent, but the great majority does use `-` between words. So I'd recommend to go with `emit-event` instead.